bin/ensemble.py

```diff
@@ -53,8 +53,7 @@ def set_file_location(args, params):
             model_dir = latest_entry_location
             
         if args.model_dir is not None:
-            target_root = '/datasets/experiments/'
-            model_dir = os.path.join(target_root, args.model_dir)
+            model_dir = args.model_dir
```

Comment:
You are probably aware of this, but anyways: 
This might requires changes in the models repo (both in the circleci script and in the release script, since they use --model_dir argument assuming the previous structure)

Reply:
Indeed, I will make a PR in models, this change was made just so we could run the release script